sdk/communication/Azure.Communication.PhoneNumbers/src/autorest.md

```diff
@@ -32,3 +32,23 @@ directive:
   transform: >
     $["x-ms-client-name"] = "twoLetterIsoCountryName";
 ```
+### Change naming of Error to ErrorMessage
+``` yaml
+directive:
+  from: swagger-document
+  where: "$.definitions.PhoneNumberSearchResult.properties.error"
+  transform: >
+    $["x-ms-enum"].name = "ErrorMessage";
+```
+
+``` yaml
+directive:
+  - remove-operation-match: /.*Reservation.*/i
+  - remove-operation: PhoneNumbers_BrowseAvailableNumbers
+  - remove-model: PhoneNumbersReservation
```

Comment:
Could we also remove the `AvailablePhoneNumber` and `AvailablePhoneNumberCost` models to avoid adding Cherry Picker models for now?